bfcfg

```diff
@@ -1758,6 +1759,52 @@ sb_cfg()
   fi
 }
 
+#
+# Dump configurations stored in EFI variable needed by OS applications
+#
+os_cfg_dump()
+{
+
+  local tmp_file=${TMP_DIR}/.bfcfg-osfs-data
+
+  if [ ${dump_level} -eq ${DUMP_LEVEL_NONE} ]; then
+    return
+  fi
+
+  if [ ! -e "${os_cfg_sysfs}" ]; then
+    log_msg "os-cfg: failed to find the ${os_cfg_sysfs} EFI variable"
```

Comment:
Let's not have it as a requirement; it is harmless to not find the variable (I think). We should simply return.

Reply:
After below comment, this is no longer valid.